multicluster/test/e2e/service_test.go

```diff
@@ -237,6 +241,49 @@ func (data *MCTestData) testStretchedNetworkPolicy(t *testing.T) {
 	assert.Equal(t, antreae2e.Dropped, connectivity, getStretchedNetworkPolicyErrorMessage(eastRegularClientName))
 }
 
+func (data *MCTestData) testStretchedNetworkPolicyReject(t *testing.T) {
+	westExpSvcInEast, err := data.getService(eastCluster, multiClusterTestNamespace, mcWestClusterTestService)
+	if err != nil {
+		t.Fatalf("Error when getting the imported Service %s: %v", mcWestClusterTestService, err)
+	}
+	westExpSvcInEastIP := westExpSvcInEast.Spec.ClusterIP
+
+	eastGwClientName := getClusterGatewayClientPodName(eastCluster)
+	eastRegularClientName := getClusterRegularClientPodName(eastCluster)
+
+	acnpBuilder := &e2euttils.ClusterNetworkPolicySpecBuilder{}
+	acnpBuilder = acnpBuilder.SetName("drop-client-pod-sel").
+		SetPriority(1.0).
+		SetAppliedToGroup([]e2euttils.ACNPAppliedToSpec{{PodSelector: map[string]string{"app": "nginx"}}}).
+		AddStretchedIngressRule(map[string]string{"app": "client"}, nil, "", nil, crdv1alpha1.RuleActionReject)
+	if _, err := data.createOrUpdateACNP(westCluster, acnpBuilder.Get()); err != nil {
+		t.Fatalf("Error creating ACNP %s: %v", acnpBuilder.Name, err)
+	}
+	defer data.deleteACNP(westCluster, acnpBuilder.Name)
+
+	// Test server Pods on the gateway Node.
+	if err := createPodWrapper(t, data, westCluster, multiClusterTestNamespace, testServerPod+"-gw", data.clusterGateways[westCluster], nginxImage, "nginx", nil, nil, nil, nil, false, nil); err != nil {
+		t.Fatalf("Error when creating nginx Pod in cluster %s: %v", westCluster, err)
+	}
+	deletePodWrapper(t, data, westCluster, multiClusterTestNamespace, testServerPod)
```

Comment:
I don't get here, why do you need to delete a `testServerPod` Pod here?

Reply:
We are not sure which Node the `testServerPod` is assigned. So I delete it and test cases that server Pod is assigned on both gw Node and worker Node.

Reply:
Sure.